runtime/src/snapshot_utils.rs

```diff
@@ -1668,43 +1668,20 @@ pub fn snapshot_bank(
 }
 
 /// Get the snapshot storages for this bank and snapshot_type
```

Comment:
Sorry I missed this earlier:

```suggestion
/// Get the snapshot storages for this bank
```